### ISSUE TITLE
dep-graph always generates JSON, no HTML or SVG, when run with file argument

### ISSUE THREAD

**Issue by GregorSondermeier** [nrwl/nx#2005]
## Expected Behavior

When specifying an output file for the dependency graph generation I expect the dep-graph to be generated in the format that is indicated by the output file's extension.

## Current Behavior

Always a JSON representation of the dep-graph is produced, regardles of the output file extension.

## Failure Information (for bugs)

It worked like expected at least until v8.1.0. Since we upgraded our nrwl dependency to 8.7.0, this false behavior appeared.  
Judging by the past issues and the GIT history of [dep-graph.js](https://github.com/nrwl/nx/blob/master/packages/workspace/src/command-line/dep-graph.ts), it looks like some heavy refactoring has been done to allow for new features, which is great. However, this one thing got lost in the process it seems.

### Steps to Reproduce

1. Run for example `npm run dep-graph -- --file=dep-graph.html`
2. The _dep-graph.html_ is generated
3. However it contains a JSON structure and not the cool and useful graphical representation of the dependency graph.

However, if you just run `npm run dep-graph` (without the `file` argument) to serve the dependency graph immediately, it is correctly represented as an HTML file containing a graphical representation and interactivity. So functionality is still there.

### Context

```
  @nrwl/angular : 8.7.0
  @nrwl/cli : 8.7.0
  @nrwl/cypress : 8.7.0
  @nrwl/eslint-plugin-nx : Not Found
  @nrwl/express : Not Found
  @nrwl/jest : 8.7.0
  @nrwl/linter : Not Found
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : Not Found
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 8.7.0
  @nrwl/web : Not Found
  @nrwl/workspace : 8.7.0
  typescript : 3.4.5
```

## Other

We want to deploy the dep-graph with our project, that's why we need it exported to a static file instead of being served immediately.

**Comment by FrozenPandaz** [comment]
@vsavkin Can you please look into adding support for generating `html` and `svg` options for the new `dep-graph`?

**Comment by benediktvaldez** [comment]
@GregorSondermeier this issue might have been resolved with #3049.

**Comment by philipjfulcher** [comment]
#3049 adds HTML, we cna keep this issue open until we have an image output as well
